### DISCUSSION TITLE
How to simulate a real clinical treatment plan

### DISCUSSION THREAD

**Discussion by JamsNeroc** [nrc-cnrc/EGSnrc#1150] [Questions] [answered]
Hello everyone.

I’m glad that I can extract the control points information in RTplan (Dicom format) and successfully make a single field simulation of a treatment plan. However, I’m confuse that how to simulate a real clinical treatment plan, which usually contains multiple fields. Here is a solution I'm thinking up naively.

For example, a plan contains two fields and the weights are w1 and w2, respectively. First, simulate each field independently and then multiply each dose by the weight and add it together using Python or something else (the main detail is that add each term in the 5th row of the 3ddose file). For this case, the final plan dose should be 3ddose1*w1 + 3ddose2*w2. I'm wondering that is there any ways can do the above simulation easily, efficiently and correctly. And  the other question is that how do I determine how many histories can equal 1 MU in the plan.

Any help I would really appreciate.

**Comment by rtownson** [comment, marked as answer]
Yes, this is a good way to structure the simulation. Each field is best to keep as a separate simulation, then use weighting to account for the MUs for each field.

See this paper on calibrating your MC simulation for absolute dose:

https://pubmed.ncbi.nlm.nih.gov/16177516/

Note that backscatter corrections may be helpful. For Truebeam they are close to 1 so it probably OK to neglect:

https://pubmed.ncbi.nlm.nih.gov/24487824/

**Reply by JamsNeroc** [reply]
Thanks for the reply. 
It seems that my thinking is correct.
The papers you mentioned were very useful and I hope I can make more breakthroughs myself.
